pkg/scheduler/framework/plugins/nodeports/node_ports_test.go

```diff
@@ -163,9 +175,14 @@ func TestPreFilterDisabled(t *testing.T) {
 	p, _ := New(nil, nil)
 	cycleState := framework.NewCycleState()
 	gotStatus := p.(framework.FilterPlugin).Filter(context.Background(), cycleState, pod, nodeInfo)
-	wantStatus := framework.AsStatus(fmt.Errorf(`reading "PreFilterNodePorts" from cycleState: %w`, fmt.Errorf("not found")))
-	if !reflect.DeepEqual(gotStatus, wantStatus) {
-		t.Errorf("status does not match: %v, want: %v", gotStatus, wantStatus)
+	wantStatus := framework.AsStatus(fmt.Errorf(`reading "PreFilterNodePorts" from cycleState: %w`, errors.New("not found")))
+	if gotStatus.AsError().Error() != wantStatus.AsError().Error() {
+		t.Error("err does not match")
+		return
+	}
+	if gotStatus.Message() != wantStatus.Message() {
+		t.Error("message does not match")
+		return
```

Comment:
Why do we need to change here? The original one looks simpler

Reply:
I have simplified it with [a8da337](https://github.com/kubernetes/kubernetes/pull/115318/commits/a8da3378a5c0d8006f95e312f1bab73aaccc87d7).